hbogolib/handlereu.py

```diff
@@ -411,7 +411,16 @@ def OAuthLogin(self, username, password):
             jsonrspl = response.json()
 
             token = jsonrspl['Token']
-            backuri = self.API_HOST_REFERER + "/ssocallbackhandler?ssoid={0}&method={1}&cou=POL&operatorId=" + self.op_id + "&p=" + self.API_PLATFORM + "&l=" + self.LANGUAGE_CODE + "&cb=" + Util.base64enc(token) + "&t=signin"
+            backuri = "%s/ssocallbackhandler?%s" % (self.API_HOST_REFERER, urlencode({
+                'ssoid': '{0}',
+                'method': '{1}',
+                'cou': 'POL',
```

Comment:
I made an error here long time ago. This is the login procedure used for "login redirect" operators. Atm work only for Cyfrowy Polsat. 
The error i made is in this line 'cou': 'POL', i left POL hard coded this should be substituted with 'cou': self.COUNTRY_CODE_SHORT . Wonder if this will fix the login for some login redirect operators. ISSUE #5

Reply:
Changed it to `self.COUNTRY_CODE_SHORT`